### ISSUE TITLE
Formatting options

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#71]
Would it be possible to add some formatting options? Here are some examples of what I'd like differently:

```
if count!=2 
vs.
if count != 2

self.setError(XUMyError, andLogObject:parsedSource)
vs.
self.setError(XUMyError, andLogObject: parsedSource) // Notice the space *after* the colon
```

**Comment by drkameleon** [comment]
I know what you mean - and that's one the (several) reasons I decided to put the Preferences panel there. It's still a bit basic, but there's sure room for such options.

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 1.9. *(I decided not to make it an extra option, but rather make it the default output format - matter of taste? perhaps. but this one might be a rather... universal taste... lol)*

**For now:**

- Space after colons (in function calls) is being taken care of
- `>`, `<`, `>=`, `<=`, `==`, `!=` are being spaced out

If you notice anything else missing, feel free to let me know.

----

**Example:**

```objective-c
@implementation aClass

- (void)someFunc {
    if (count!=2) {
        [self setError:XUMyError andLogObject:parsedSource];
    }
}
@end
```

**Output:**

```swift
class aClass {

	func someFunc() {
		if count != 2 {
			self.setError(XUMyError, andLogObject: parsedSource)
		}
	}
	
}
```
